docs/user_guide/method/method.rst

```diff
@@ -0,0 +1,2330 @@
+    coupling_MX = Coupling(site_index=[1, 2], isotropic_j=12)
+    couplings = [coupling_AM, coupling_AX, coupling_MX]
+    system = SpinSystem(sites=sites, couplings=couplings)
+
+    method = Method(
+        channels=["1H"],
+        magnetic_flux_density=9.4,  # in T
+        spectral_dimensions=[
+            SpectralDimension(
+                count=16000,
+                spectral_width=1800,  # in Hz
+                reference_offset=1000,  # in Hz
+                label="$^{1}$H frequency",
+                events=[
+                  {
+                  "fraction":1,
+                  "transition_query":[{"ch1":{"P":[-1]}}]
+                  }
+                ]
+            )
+        ]
+    )
+
+    sim = Simulator(spin_systems = [system],methods=[method])
+    sim.run()
+
+    processor = sp.SignalProcessor(
+        operations=[sp.IFFT(),sp.apodization.Exponential(FWHM="1 Hz"),sp.FFT()]
+    )
+
+    plt.figure(figsize=(10, 3))  # set the figure size
+    ax = plt.subplot(projection="csdm")
+    ax.plot(processor.apply_operations(dataset=sim.methods[0].simulation))
+    ax.invert_xaxis()  # reverse x-axis
+    plt.tight_layout()
+    plt.grid()
+    plt.show()
+
+The assignment of transitions in the spectrum above are, from left to right, are
+:math:`\hat{X}_4, (\hat{X}_3, \hat{X}_2)`, and :math:`\hat{X}_1` centered at 4.5
+ppm, :math:`\hat{M}_4, (\hat{M}_3, \hat{M}_2)`, and :math:`\hat{M}_1` centered
+at 2.5 ppm, and :math:`\hat{A}_4, (\hat{A}_3, \hat{A}_2)`, and :math:`\hat{A}_1`
+centered at 0.5 ppm.
+
+It is essential to realize that all Sites having the same isotope are
+"indistinguishable" to a TransitionQuery object. Recall that ``ch1`` is
+associated with the first isotope in the list of isotope strings assigned to the
+Method attribute ``channels``. When the TransitionQuery above is combined with
+the SpinSystem object with three :math:`^1\text{H}` Sites, it must first expand
+its SymmetryQuery into an intermediate set of spin-system-specific symmetry
+queries, illustrated by each row in the table below.
+
+.. list-table::
+   :widths: 25 25 25 25
+   :header-rows: 1
+
+   * - Transitions
+     - :math:`\text{p}_A`
+     - :math:`\text{p}_M`
+     - :math:`\text{p}_X`
+   * - :math:`\hat{A}_1, \hat{A}_2, \hat{A}_3, \hat{A}_4`
+     - –1
+     - 0
+     - 0
+   * - :math:`\hat{M}_1, \hat{M}_2, \hat{M}_3, \hat{M}_4`
+     - 0
+     - –1
+     - 0
+   * - :math:`\hat{X}_1, \hat{X}_2, \hat{X}_3, \hat{X}_4`
+     - 0
+     - 0
+     - –1
+
+The intermediate spin-system-specific symmetry query in each row selects a
+subset of transitions from the complete set of transitions. The final set of
+selected transitions is obtained from the union of transition subsets from each
+spin-system-specific symmetry query.
+
+The :py:meth:`~mrsimulator.Method.get_transition_pathways` function will allow
+you to inspect the transitions selected by the TransitionQuery objects in the
```

Comment:
@pjgrandinetti This phrasing suggests the ``get_transition_pathways`` function is associated with a SpectralEvent, not Method (I know users could click on the api ref link, but still). Thoughts on changing to 

... function will allow you to inspect the transitions selected by a **Method** object terms of the initial and final Zeeman eigenstate quantum numbers. Which transitions a **Method** object selects depends on which **SpectralEvents** are in its spectral dimensions and what **TransitionQuery** objects those **SpectralEvents** hold.

Reply:
I suggest just replacing "the TransitionQuery objects in the SpectralEvent" with "a Method", i.e.,

"The get_transition_pathways() function will allow you to inspect the transition pathways selected by a Method in terms of the initial and final Zeeman eigenstate quantum numbers. "

Reply:
Maybe there should also be a "def get_transitions(spin_system)" function for spectral and delay event.

Reply:
I suppose that could be helpful if the Method had a lot of events.   Do we have any examples where it could be useful?